relayer/chain/relaychain/connection.go

```diff
@@ -292,7 +218,8 @@ func (co *Connection) fetchParaHeads(blockHash types.Hash) (map[uint32]ParaHead,
 		return nil, err
 	}
 
-	heads := make(map[uint32]ParaHead)
+	const numParas = 16
```

Comment:
Where does this number come from?

Reply:
Just initial compacity of the array which can be extended as required.